react-graphql-client-library.md

```diff
@@ -23,31 +23,31 @@ summary = "A tutorial on how to build your own GraphQL client for React applicat
-* see how you would be able to contribute to the GraphQL ecosystem
+* how a GraphQL client library works under the hood
+* how simple a GraphQL client library is to implement your way
+* how it contributes to the GraphQL ecosystem
 
-I believe there is great potential to contribute to the GraphQL ecosystem, because it is fairly new. It would be great to have a diverse set of tools out there and not only one ecosystem pushing the whole thing forward. So see it as your opportunity to contribute to it.
+There is contribute to the GraphQL ecosystem, because the tools surrounding it are still fairly new. A diverse set of tools would speed this along insgtead of the one ecosystem pushing its maturation forward. This is not only a useful addition for your web development skillset, it is alos an opportunity to contribute to the early stages of a software's growth.
 
-Before diving into the single parts of implementing your own GraphQL client for React, let's think about the essential parts to be able to consume a GraphQL API in your React application?
+Before diving into implementing your own GraphQL client for React, consider the essentials for consuming a GraphQL API in a React application:
 
-* First, there must be **the GraphQL client itself**. It can be any HTTP library or even the {{% a_blank "native fetch API" "https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API" %}}, but it must be able to send HTTP methods with a payload across the wire. Note: Even though the GraphQL specification isn't opinionated about the transportation layer, the GitHub GraphQL API (which you are going to consume with your own GraphQL client) is using HTTP. Thus our GraphQL client has to be able to execute GraphQL operations by using HTTP methods.
+1. The **GraphQL client** must be used. It can be any HTTP library or even the {{% a_blank "native fetch API" "https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API" %}}, but it must be able to send HTTP methods with a payload across the wire.  While the GraphQL specification isn't opinionated about the transportation layer, the GitHub GraphQL API you  consume with GraphQL client is using HTTP. Our GraphQL client must be able to execute GraphQL operations by using HTTP methods.
 
-* Second, there must be a way to **provide the GraphQL client instance to the React view layer**. It is the perfect use case for [React's Context API](https://www.robinwieruch.de/react-context-api/) to provide the GraphQL client instance somewhere at the top level of your React component tree and to consume it in every React component which is interested in it.
+2. There must be a way to **provide the GraphQL client instance to the React view layer**. It is the perfect use for [React's Context API](https://www.robinwieruch.de/react-context-api/) to provide the GraphQL client instance at the top level of the React component tree, and to consume it in every React component interested in it.
```

Comment:
"Second," instead of "2."

Reply:
See previous notes.  Changed to bulleted list, removed enumeration.